kafka/setup_scripts/topics.txt

```diff
@@ -1 +1,2 @@
-timecard-timeentries
\ No newline at end of file
+timecard-timeentries
```

Comment:
can the old topic name be removed now?

Reply:
I've removed and taken out the related permissions as well